src/tools/react-native.ts

```diff
@@ -327,27 +327,19 @@ export function updateExpoRouterSrcDir(toolbox: GluegunToolbox) {
   })
 }
 
-export async function cleanupExpoRouterConversion(toolbox: GluegunToolbox) {
-  const { system, parameters, print } = toolbox
-
-  // debug?
-  const debug = boolFlag(parameters.options.debug)
-  const log = <T = unknown>(m: T): T => {
-    debug && print.info(` ${m}`)
-    return m
-  }
+export function cleanupExpoRouterConversion(toolbox: GluegunToolbox, targetPath: string) {
```

Comment:
I think this section still needs some work, I'm getting a failure when I run the Expo Router workflow

```
npx ignite-cli new ExpoRouterApp \
        --bundle=com.exporouterapp \
        --git \
        --install-deps=false \
        --packager=npm \
        --target-path=C:\Users\joshu\Code\ExpoRouterApp \
        --remove-demo=false \
        --experimental=expo-router \
        --workflow=CNG \
        --no-timeout=false \
        --state=mst
```

When I try to run any expo commands, I get

```
CommandError: Cannot find module 'ajv/dist/compile/codegen'
Require stack:
- C:\Users\joshu\Code\ExpoRouterApp\node_modules\ajv-keywords\dist\definitions\typeof.js
```